include/uart.h

```diff
@@ -125,6 +182,10 @@ struct uart_driver_api {
 	/** Console I/O function */
 	int (*err_check)(struct device *dev);
 
+	/** UART configuration functions */
+	int (*configure)(struct device *dev, struct uart_config *cfg);
```

Comment:
It should be `const struct uart_config *cfg`.

Reply:
fixed

Reply:
fixed